packages/renderer/src/lib/preferences/PreferencesProviderRendering.svelte

```diff
@@ -63,6 +63,24 @@ async function startReceivinLogs(provider: ProviderInfo): Promise<void> {
 async function stopReceivingLogs(provider: ProviderInfo): Promise<void> {
   await window.stopReceiveLogs(provider.internalId);
 }
+
+async function consilienceCheck(internalProviderId: string, params: { [key: string]: any }): Promise<boolean> {
+  const checksToReview = await window.createKubernetesProviderConsilienceCheck(internalProviderId, params);
+
+  if (checksToReview) {
+    let message = checksToReview.join('\n');
+    let type = 'warning';
+    const answer = await window.showMessageBox({
+      title: 'Create a Connection',
+      type: type,
+      message: message,
+      buttons: ['Create Anyway', 'Cancel'],
```

Comment:
```suggestion
      buttons: ['Continue', 'Cancel'],
```

Maybe continue instead of Create Anyway?

Reply:
reworked with displaying messages instead of modal dialog